apps/condo/domains/division/components/DivisionsTable.tsx

```diff
@@ -67,75 +72,83 @@ export default function DivisionTable (props: BuildingTableProps) {
     }
 
     const [search, handleSearchChange] = useSearch<IFilters>(loading)
+    const isNoDivisionsData = !divisions.length && isEmpty(filters) && !loading
```

Comment:
When wi will have empty divisions list for some filter query, a stub will not be displayed, according to this logic.
Are you sure that this is correct? Ask design team.